### DISCUSSION TITLE
Livros free da O'reilly

### DISCUSSION THREAD

**Discussion by WandersonAlves** [frontendbr/forum#582] [Dicas]
A O'Reilly liberou alguns dos seus livros de forma gratuita http://www.oreilly.com/free/reports.html

Fica a dica ai!

**Comment by sergiohampel** [comment]
Boa, Popoto!!! \m/

**Comment by WandersonAlves** [comment]
Opa @sergiohampel quanto tempo kkkkkkkkkkk

**Comment by fernandomoraes** [comment]
Bom demais, baixei quase tudo lá (Y)
